airflow/www/templates/airflow/tree.html

```diff
@@ -20,6 +20,11 @@
 {% extends "airflow/dag.html" %}
 {% block page_title %}{{ dag.dag_id }} - Tree - {{ appbuilder.app_name }}{% endblock %}
 
+{% block head_meta %}
+  {{ super() }}
+  <meta name="blur" content="{{ blur }}">
```

Comment:
We should really change this variable name to something more descriptive like `is_demo_mode` (in a separate PR).